crates/web_server/src/lib.rs

```diff
@@ -37,39 +45,43 @@ struct WebServerState<KEY> {
     latest_quorum_proposal: u64,
     /// view for the most recent view sync proposal
     latest_view_sync_proposal: u64,
-
-    /// view for teh oldest DA certificate
+    /// view for the oldest DA certificate
     oldest_certificate: u64,
-
+    /// view for the oldest view sync certificate
     oldest_view_sync_proposal: u64,
     /// view number -> Vec(index, vote)
     votes: HashMap<u64, Vec<(u64, Vec<u8>)>>,
-
+    /// view sync: view number -> Vec(index, vote)
     view_sync_votes: HashMap<u64, Vec<(u64, Vec<u8>)>>,
     /// view number -> highest vote index for that view number
     vote_index: HashMap<u64, u64>,
-
+    /// view_sync: view number -> highest vote index for that view number
     view_sync_vote_index: HashMap<u64, u64>,
     /// view number of oldest votes in memory
     oldest_vote: u64,
-
+    /// view sync: view number of oldest votes in memory
     oldest_view_sync_vote: u64,
-
+    /// view number -> (secret, proposal)
+    /// TODO is this right?
     vid_disperses: HashMap<u64, (String, Vec<u8>)>,
+    /// view for the oldest vid disperal
     oldest_vid_disperse: u64,
+    /// view of most recent vid dispersal
     recent_vid_disperse: u64,
-
+    /// TODO document, not sure what this is
     vid_votes: HashMap<u64, Vec<(u64, Vec<u8>)>>,
+    /// oldest vid vote view number
     oldest_vid_vote: u64,
-    // recent_vid_vote: u64,
+    /// recent_vid_vote view number
     vid_certificates: HashMap<u64, (String, Vec<u8>)>,
+    /// oldest vid certificate view number
     oldest_vid_certificate: u64,
-    // recent_vid_certificate: u64,
+    /// recent_vid_certificate: u64,
     vid_vote_index: HashMap<u64, u64>,
-
     /// index -> transaction
     // TODO ED Make indexable by hash of tx
     transactions: HashMap<u64, Vec<u8>>,
+    /// TODO document
```

Comment:
This was supposed to be a mapping of tx hash to tx index, but I don't think it is actually used.